### PR TITLE
Toggling visibility binds/unbinds mouse handlers

### PR DESCRIPTION
**Pull request by jbeezley**
Addresses #387

### CODE REVIEW

**Comment by jbeezley** [comment]
Hey look, buildbot status. :trophy:

**Comment by aashish24** [comment]
Nice!! :clap:

File: src/core/feature.js [hunk 1]
```diff
@@ -378,6 +378,14 @@ geo.feature = function (arg) {
     } else {
       m_visible = val;
       m_this.modified();
+
+      // bind or unbind mouse handlers on visibility change
+      if (m_visible) {
```
Comment (aashish24) [inline src/core/feature.js]: +1

**Comment by aashish24** [comment]
LGTM :+1:

**Comment by aashish24** [comment]
merging it. @manthey  can reopen it if problem still exists.

**Pull request merged**